javascript/newer-JS-features.md

```diff
@@ -222,6 +222,184 @@ const { email, password } = user; //equivalent to saying const email = user.emai
 console.log(email); // renunikhilp@gmail.com
 ```
 
+### Temporal API Features
+
+Temporal API provides better date and time handling capabilities in JavaScript. We'll look at the classes: `PlainDate`, `PlainTime`, and `ZonedDateTime`.
+
+### **PlainDate**
```

Comment:
```suggestion
##### **PlainDate**
```

Reply:
I've added this along with some additional formatting updates in a commit

Reply:
It’s best practice to cite which commit by providing your hash in the replying comment.